### PR TITLE
WIP - [ActionSheet] Manual layout for action cells.

### PR DESCRIPTION
**Pull request by romoore**
Switches the cells to use manual layout instead of auto layout. This will
simplify the view hierarchy and complexity of constraints for upcoming edge
insets APIs.

### CODE REVIEW

File: components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m [hunk 1]
```diff
@@ -130,29 +82,117 @@ - (void)commonMDCActionSheetItemViewInit {
...
-                                             constant:kImageTopPadding]
-      .active = YES;
-  [_actionImageView.leadingAnchor constraintEqualToAnchor:_contentContainerView.leadingAnchor]
-      .active = YES;
-  [_actionImageView.widthAnchor constraintEqualToConstant:kImageHeightAndWidth].active = YES;
-  [_actionImageView.heightAnchor constraintEqualToConstant:kImageHeightAndWidth].active = YES;
+  [self.contentView addSubview:_actionImageView];
+}
+
+- (UIEdgeInsets)_contentPadding {
```
Comment (romoore) [inline components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m]: Go away

**Comment by romoore** [comment]
Internal testing started: cl/294471580

File: components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m (hunk 1 above)
Comment (romoore) [reply components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m]: Done.

File: components/BottomSheet/src/private/MDCSheetContainerView.m [hunk 2]
```diff
@@ -84,6 +84,7 @@ - (instancetype)initWithFrame:(CGRect)frame
                  forKeyPath:kContentSizeKey
                     options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld
                     context:kContentSizeContext];
+    [scrollView addObserver:self forKeyPath:@"contentInset" options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld context:kContentSizeContext];
```
Comment (romoore) [inline components/BottomSheet/src/private/MDCSheetContainerView.m]: Clean-up. Commit separately but justify the change.

**Review by randallli — DISMISSED** [review]

**Comment by randallli** [comment]
You should be able to sync to head to get some of the already failing tests to pass now.

Also CI is complaining about clang-format

**Comment by romoore** [comment]
Thanks, Randall. This PR isn't ready at the moment. I'm working with @codeman7 to land the BottomSheet changes separately in #9726. For that other PR, I plan to wait until @yarneo is back to work on Tuesday next week so he has a chance to review.

**Comment by jverkoey** [comment]
Closing this PR for now, but please re-open if/when you are working on it again.

**Pull request closed without merging**